cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -142,7 +142,7 @@ public static KafkaConnectCluster fromCrd(KafkaConnect kafkaConnect, KafkaVersio
     protected static <C extends KafkaConnectCluster> C fromSpec(KafkaConnectSpec spec,
                                                                 KafkaVersion.Lookup versions,
                                                                 C kafkaConnect) {
-        kafkaConnect.setReplicas(spec.getReplicas() > 0 ? spec.getReplicas() : DEFAULT_REPLICAS);
+        kafkaConnect.setReplicas(spec.getReplicas() >= 0 ? spec.getReplicas() : DEFAULT_REPLICAS);
```

Comment:
replicas is an `int` in `KafkaConnectSpec`. I assume that if you created a `KafkaConnect` without a `relicas` then Kubernetes will give us the JSON with the defaulted value (3), but have you tested that? Because if it gave us a JSON with `replicas` omitted then we'd start using 0 as the default, which would be wrong.

Reply:
It gives us a default value 3.

Reply:
you are right... fixing